api.md

```diff
@@ -170,6 +172,70 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## attach_data_node() [](attach_data_node)
+
+Attach a data node to a hypertable. The data node should have been
+previously created using [`add_data_node`](#add_data_node).
+
+When a distributed hypertable is created it will by default use all
+available data nodes for the hypertable, but if a data node is added
+*after* a hypertable is created, the data node will not automatically
+be used by existing distributed hypertables.
+
+If you want a hypertable to use a data node that was created later,
+you need to attach the data node to the hypertable using this
+function.
+
+#### Required Arguments [](attach_data_node-required-arguments)
+
+| Name              | Description                                   |
+| `hypertable`      | Name of hypertable to attach node to          |
+| `node_name`       | Local name of data node to attach             |
+
+##### Parameter Description
+
+* `hypertable` - The name of the hypertable where the data node should
+  be attached.
+
+* `node_name` - Local name of the data node to attach to the
+  hypertable.
+
+* `if_not_attached` - Do not generate an error if the data node is
+  already attached to the hypertable. A notice will be printed that
+  the data node is attached.
+
```

Comment:
Cut the `if_not_attached` from above.  It's included correctly in optional and this should be removed from this required section

Reply:
Done